src/types/MetricsOptions.ts

```diff
@@ -1,5 +1,6 @@
 interface MetricsOptions {
   rateInterval?: string;
+  pollInterval?: number;
```

Comment:
IMO it shouldn't go in MetricsOptions, since this is what is sent to the server (and the server API doesn't have a `pollInterval` query param).
So, it could be a second parameter in MetricsOptionsBar `onOptionsChanged` props.

Reply:
But onOptionsChanged: (opts: MetricsOptions) => void; Is the type MetricsOptions I can't define pollInterval if I not define it in MetricOptions type.

Reply:
I'm not sure to understand, you can change the method signature in `MetricsOptionsBar.tsx` no?
Or an alternative, to really decorrelate API options versus UI options: pass another function as argument to `MetricsOptionsBar`, such as `onPollIntervalChanged`. That would be even better because there's no reason to fetch metrics immediately after changing the poll interval.

Reply:
Ok I understand now ^^